charts/icm-as/values.yaml

```diff
@@ -69,7 +69,7 @@ jvm:
     # enable/disable to java-vm-debugging
     enabled: false
   # define options / parameters to be used to start the jvm
-  options: "-XX:NewRatio=8 -XX:MaxRAMPercentage=80 -XX:InitialRAMPercentage=80"
+  options: "-XX:MaxRAMPercentage=70 -XX:InitialRAMPercentage=70"
```

Comment:
You deleted the `NewRatio` parameter. Seeing that the default value is 2 this probably leads to very different GC behavior. Do you have a reason for this change?

Reply:
Oh, after reading the Issue again I see that this was mentioned explicitly. Never mind...